[2023-03-26 20:35:56,633] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_and_backfill_v01.first_task 2023-03-18T00:00:00+00:00 [queued]>
[2023-03-26 20:35:56,664] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_and_backfill_v01.first_task 2023-03-18T00:00:00+00:00 [queued]>
[2023-03-26 20:35:56,666] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-26 20:35:56,667] {taskinstance.py:1069} INFO - Starting attempt 1 of 6
[2023-03-26 20:35:56,668] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-26 20:35:56,678] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): first_task> on 2023-03-18T00:00:00+00:00
[2023-03-26 20:35:56,686] {standard_task_runner.py:52} INFO - Started process 117 to run task
[2023-03-26 20:35:56,697] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_and_backfill_v01', 'first_task', '2023-03-18T00:00:00+00:00', '--job-id', '57', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cathcup_and_backfill.py', '--cfg-path', '/tmp/tmp0184mztt', '--error-file', '/tmp/tmpml667y5w']
[2023-03-26 20:35:56,698] {standard_task_runner.py:77} INFO - Job 57: Subtask first_task
[2023-03-26 20:35:56,789] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v01.first_task 2023-03-18T00:00:00+00:00 [running]> on host fded5fa0e87e
[2023-03-26 20:35:56,887] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mario
AIRFLOW_CTX_DAG_ID=dag_with_catchup_and_backfill_v01
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-18T00:00:00+00:00
[2023-03-26 20:35:56,890] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-03-26 20:35:56,893] {bash.py:158} INFO - Running command: echo catchup on Airflow!!!
[2023-03-26 20:35:56,912] {bash.py:169} INFO - Output:
[2023-03-26 20:35:56,954] {bash.py:173} INFO - catchup on Airflow!!!
[2023-03-26 20:35:56,957] {bash.py:177} INFO - Command exited with return code 0
[2023-03-26 20:35:57,018] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v01, task_id=first_task, execution_date=20230318T000000, start_date=20230326T203556, end_date=20230326T203557
[2023-03-26 20:35:57,054] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-26 20:35:57,079] {local_task_job.py:146} INFO - Task exited with return code 0
